### ISSUE TITLE
Google Drive Implementation

### ISSUE THREAD

**Issue by gio1135** [dart-lang/chromedeveditor#3894]
You guys should have an "Import from Drive" option or something. So you can directly work on a Google Drive directory.

**Comment by johnalexhebert** [comment]
Importing docs and folders from Google Drive intermittently works, at least on small files or folders. When I first tried to import a large folder Google Drive to a folder in my CDE project window, I get a dialog error of:

    Error while importing folder

    [object ProgressEvent]

Apparently the problem is displaying the progress bar of the data transfer. However, there is a local file sync taking place on my Chromebook.

I'll try to test and describe this behavior more fully soon and post a bug if needed.

**Comment by gio1135** [comment]
So you can directly with on Google Drive folders!? How?

**Comment by johnalexhebert** [comment]
Right-click on the project folders in CDE and select "Import Folders". Go to your Drive and open the folder containing the files and sub-folders you want to import. Click "Open". If the intermittent bug doesn't happen you should end up with a new folder and files in your project.

Here is what The Monkees "Last Train to Clarksville" looks like in CDE:
![screenshot 2015-03-25 at 00 23 55](https://cloud.githubusercontent.com/assets/3825230/6818875/fe24ccb8-d285-11e4-9427-93ceba9ff86f.png)

**Comment by gio1135** [comment]
Cool, thanks! Did you make Cheering Dev Editor?

**Comment by johnalexhebert** [comment]
Oh, no. Just another cheery Chromebook user.
And if you want to know what the USA was watching on TV before it went insane:
https://www.youtube.com/watch?v=tGLx4WenGwQ

**Comment by gio1135** [comment]
Whoa! I meant to put Chrome not Cheery but I was swiping and wasn't really
paying attention to what came out.

**Comment by gio1135** [comment]
Ooooh, I see. I'm using Ubuntu. I had two Chromebooks but then two kids
starting fighting over them and broke the screens. So, I'm wondering how
you would do it on Ubuntu.
